tests/FakeItEasy.IntegrationTests/StringArgumentConstraints.cs

```diff
@@ -1,19 +1,17 @@
 namespace FakeItEasy.IntegrationTests
 {
     using System;
-    using System.Globalization;
 
     public static class StringArgumentConstraints
     {
         public static string StartsWith(this IArgumentConstraintManager<string> scope, string beginning)
         {
-            return scope.Matches(
-                x => x.StartsWith(beginning, StringComparison.CurrentCulture), string.Format(CultureInfo.CurrentCulture, "Starts with \"{0}\"", beginning));
+            return scope.Matches(x => x.StartsWith(beginning, StringComparison.CurrentCulture), $@"Starts with ""{beginning}""");

```

Comment:
I restored the `StringComparison.CurrentCulture` because

1. I was overeager; it's not really anything to do with this issue, and
2. removing it introduced a code analysis warning that I didn't care for